api/src/main/java/io/strimzi/api/kafka/model/template/PodTemplate.java

```diff
@@ -182,6 +184,16 @@ public void setEnableServiceLinks(Boolean enableServiceLinks) {
         this.enableServiceLinks = enableServiceLinks;
     }
 
+    @Pattern("^([0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$")
+    @Description("Defines the total amount of local storage required for temporary EmptyDir volume (for example 1Gi).")
```

Comment:
I think we should mention that this is used for `/tmp`, otherwise it's users won't know what "temporary EmptyDir volume" really means. We should also mention the default value.